modules/building/pages/creating.adoc

```diff
@@ -2,29 +2,37 @@
 
 == Onboarding components to {ProductName}
 
-{ProductName} supports two methods for creating applications and components.
+{ProductName} supports two methods for creating applications and components:
 
-Before being able to onboard a component to {ProductName}, you will need to ensure that the instance has appropriate access to the git repository. This means either installing your organization’s {ProductName} GitHub App on the source code repository or xref:building:creating-secrets.adoc#creating-source-control-secrets[create a secret] to enable access to a GitLab repository.
+* With the UI
+* With the CLI
 
-NOTE: Component names must be unique in a namespace, even when components are used in different application.
+Applications in Konflux are simply logical groupings of components. So to create an application, you must first create at least one component. 
```

Comment:
I am not sure that this is technically true anymore, but it is probably fine to leave it.